src/UNL/UCBCN/ActiveRecord/Record.php

```diff
@@ -283,13 +283,12 @@ public static function __callStatic($method, $args)
             }
 
             return self::getByAnyField($class, $field, $args[0], $whereAdd);
-
+            break;
         }
         throw new Exception('Invalid static method called.', 500);
     }
 
-    public static function getByAnyField($class, $field, $value, $whereAdd = '')
-    {
+    public static function getByAnyField($class, $field, $value, $whereAdd = '') {
```

Comment:
Very well. I actually preferred doing it this way back in college, but my early employers stressed, and convinced me, that this was a waste of a line of code. I will defer to the PSR though.